ci-operator/step-registry/acm/fetch-managed-clusters/acm-fetch-managed-clusters-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+# Set the dynamic vars based on provisioned hub cluster.
+HUB_OCP_API_URL=$(oc whoami --show-server)
+export HUB_OCP_API_URL
+HUB_OCP_PASSWORD=$(cat $SHARED_DIR/kubeadmin-password)
+export HUB_OCP_PASSWORD
```

Comment:
Why not just do `export HUB_OCP_PASSWORD=$(cat $SHARED_DIR/kubeadmin-password)` ?

Reply:
same as above